SignalMessaging/environment/OWSSounds.m

```diff
@@ -224,7 +224,7 @@ - (void)playSound:(OWSSound)sound quiet:(BOOL)quiet shouldRespectSilentSwitch:(B
     [self.audioPlayer stop];
     self.audioPlayer = [OWSSounds audioPlayerForSound:sound quiet:quiet];
     if (shouldRespectSilentSwitch) {
-        [self.audioPlayer playWithCurrentAudioCategory];
+        [self.audioPlayer playWithAmbientAudioCategory];
```

Comment:
This is a regression from 2.20. 

Previously we used `AudioServicesPlayAlertSound` which had behavior like Ambient+"ducks others"

At first I took a stab at restoring the usage of AudioServicesPlayerAlertSound, but I think actually the *ducks others* behavior is still pretty distracting and not necessary (e.g. see WA).